extensions/sql-migration/src/dialog/assessmentResults/assessmentResultsDialog.ts

```diff
@@ -46,7 +46,8 @@ export class AssessmentResultsDialog {
 					}).component();
 					flex.addItem(await this._tree.createRootContainer(view), { flex: '1 1 auto' });
 
-					view.initializeModel(flex);
+					await view.initializeModel(flex);
+					await this._tree.focusComponent.focus();
```

Comment:
The modal dialogs are supposed to pick the first tabbable element on a page when they're shown - @aasimkhan30 could you work with Brian to figure out if there's a bug with the base implementation of that logic? Ideally we shouldn't need every person creating a dialog to manually set the focus unless they want to focus something besides the first tabbable element on a page

https://github.com/microsoft/azuredatastudio/blob/main/src/sql/workbench/browser/modal/modal.ts#L384